portability-types-transfer/src/main/java/org/dataportabilityproject/types/transfer/auth/TokenSecretAuthData.java

```diff
@@ -0,0 +1,28 @@
+package org.dataportabilityproject.types.transfer.auth;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+
+/** Token-secret-based authentication data. */
+@JsonTypeName("org.dataportability:TokensAndUrlAuthData")
```

Comment:
I think the json name needs to change too.

Reply:
woops - copy paste error